### ISSUE TITLE
Silent failure when trying to require missing file

### ISSUE THREAD

**Issue by dgtombs** [bertramdev/grails-asset-pipeline#355]
Steps to reproduce:

1. Try to require a file that does not exist, e.g., `//= require ./doesnotexist.js`

Expected Result:

Asset-pipeline fails the build or at least outputs a warning about the missing file.

Actual Result with asset-pipeline v2.9.1:

There is no warning. The requirement is silently dropped.

**Comment by thojo** [comment]
Should optionally be possible so that missing files would be noticed immediately.

**Comment by stefanrother** [comment]
Definitely. I'd like to put this as a step into our Continuous Delivery Pipeline.
